roles/keystone/meta/main.yml

```diff
@@ -17,3 +17,5 @@ dependencies:
     when: openstack_install_method == 'package'
   - role: collectd-plugin
     when: collectd is defined and collectd.enabled|bool
+  - role: openldap
```

Comment:
I dont think you want this.  This will make openlap a dependency of keystone and breaks role composability.  The better way to do this would be to add this to site.yml for:
```
hosts: openldap
```